src/main/java/uk/gov/hmcts/reform/blobrouter/data/notifications/NotificationEnvelopeRepository.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.reform.blobrouter.data.notifications;
+
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public class NotificationEnvelopeRepository {
+
+    private final NamedParameterJdbcTemplate jdbcTemplate;
+    private final NotificationEnvelopeMapper mapper;
+
+    public NotificationEnvelopeRepository(
+        NamedParameterJdbcTemplate jdbcTemplate,
+        NotificationEnvelopeMapper mapper
+    ) {
+        this.jdbcTemplate = jdbcTemplate;
+        this.mapper = mapper;
+    }
+
+    public List<NotificationEnvelope> getRejectedEnvelopes() {
+        return jdbcTemplate.query(
+            "SELECT env.file_name, env.container, event.type as error_type, event.notes as description "
+                + " FROM envelopes env, envelope_events event "
+                + " WHERE env.id = event.envelope_id "
+                + " AND event.type = 'REJECTED'",
```

Comment:
looks like `error_type` is always 'REJECTED' then

Reply:
do we need a new model, or can this just return normal envelopes?

Reply:
Envelope doesn't have `notes`, which is mapped to `error_description` in the `NotificationMsg`

Reply:
ok

Reply:
`error_type` field in the model is still not needed though, right?

Reply:
removed `error_type`